jwst/associations/lib/asn_schema_jw_level2b.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://json-schema.org/schema#",
```

Comment:
This causes jsonschema to use the latest draft, whatever that is, which is a deprecated behavior in jsonschema 3.0.  Also, the latest draft is now draft-07, whereas in jsonschema 2.6 it was draft-04 anyway.

Reply:
ping @stscieisenhamer